sql/telemetry/firefox_kpi_dashboard_v1/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: Firefox KPI dashboard
+description: Firefox KPI dashboard
+owners:
+  - jklukas@mozilla.com
+labels:
+  incremental: true
+  schedule: 45 15 * * *
```

Comment:
This is actually not valid a label.

Reply:
Alternatively, it might be sufficient to define `schedule` just in the DAG metadata

Reply:
Yes, I think it makes sense to have schedule be defined only in the DAG metadata